bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/link/LinkImplTest.java

```diff
@@ -19,25 +19,32 @@
 
 import com.adobe.cq.wcm.core.components.context.CoreComponentTestContext;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import com.adobe.cq.wcm.core.components.commons.link.Link;
 import com.day.cq.wcm.api.Page;
 import com.google.common.collect.ImmutableMap;
 
+import io.wcm.testing.mock.aem.junit5.AemContext;
+import io.wcm.testing.mock.aem.junit5.AemContextExtension;
+
 import static com.adobe.cq.wcm.core.components.internal.link.LinkImpl.*;
 import static com.adobe.cq.wcm.core.components.internal.link.LinkTestUtils.assertInvalidLink;
 import static com.adobe.cq.wcm.core.components.internal.link.LinkTestUtils.assertValidLink;
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.mock;
 
+@ExtendWith(AemContextExtension.class)
 class LinkImplTest {
 
     private static final String URL = "/url.html";
+    
+    private final AemContext context = CoreComponentTestContext.newAemContext();
```

Comment:
Could we avoid passing the context in all assertValidLinks? I see the context is instantiated here, maybe we can do that in LinkTestUtils?

Reply:
``assertValidLink`` is also used by other test classes; for example in ``LInkManagerTest`` I see that the context is also used in other ways. I haven't investigated if these tests would also work with 2 different instances of the AemContext, but I also don't think that it's worth the effort.